doc/manual/src/command-ref/env-common.md

```diff
@@ -44,6 +44,16 @@ Most Nix commands interpret the following environment variables:
     The Nix search path can also be extended using the `-I` option to
     many Nix commands, which takes precedence over `NIX_PATH`.
 
+    If `NIX_PATH` is not set at all, Nix will fall back to the following list in impure and unrestricted evaluation mode:
+
+      1. `$HOME/.nix-defexpr/channels`
+      2. `nixpkgs=/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
+      3. `/nix/var/nix/profiles/per-user/root/channels`
+
+    If `NIX_PATH` is set to an empty string the above fallback will not happen and building will fail with:
+
+      `error: file 'nixpkgs' was not found in the Nix search path`
```

Comment:
```suggestion
Thanks for the quick response. Trying to be even more sharp:
```suggestion
    If `NIX_PATH` is set to an empty string, resolving search paths will always fail.
    For example, attempting to use `<nixpkgs>` will produce:

        error: file 'nixpkgs' was not found in the Nix search path
```

The error message should now render as code.